runtime/queries/blade/injections.scm

```diff
@@ -0,0 +1,18 @@
+((text) @injection.content
+    (#not-has-ancestor? @injection.content "envoy")
+    (#set! injection.combined)
+    (#set! injection.language php))
+
+; could be bash or zsh
+; or whatever tree-sitter grammar you have.
+((text) @injection.content
+    (#has-ancestor? @injection.content "envoy")
+    (#set! injection.combined)
+    (#set! injection.language bash))
```

Comment:
`#has-ancestor?` / `#not-has-ancestor?` is a predicate that only exists in nvim as far as I can tell. For now we can remove that line and always inject `php`